tests/testsuite.src/run_misc.at

```diff
@@ -1172,6 +1172,32 @@ AT_CHECK([$COBCRUN_DIRECT ./prog2], [1], [],
 AT_CLEANUP
 
 
+AT_SETUP([Non-numeric data in numeric items (lax checks)])
```

Comment:
That is not "lax check" but "MOVE of invalid data in numeric item to alphanumeric" or similar like that